### ISSUE TITLE
Multiple sections can be expanded with Bootstrap 5.1 accordion

### ISSUE THREAD

**Issue by ShaCP** [twbs/bootstrap#35685]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

I noticed it works differently compared to 5.0. If you quickly press two buttons it will open both of them. 5.0 correctly blocks the second one. You can check this on the official docs demos.

### Reduced test cases

Official doc demos:
https://getbootstrap.com/docs/5.1/components/accordion/#example
https://getbootstrap.com/docs/5.0/components/accordion/#example

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Bootstrap are you using?

5.1

**Comment by WinterSilence** [comment]
problem in `Collapse._isTransitioning`. By default, `_isTransitioning` is false and user can toggle second item before first transition is completed

**Comment by GeoSot** [comment]
Nice catch guys.

The problem seems to be there

`
const SELECTOR_ACTIVES = '.collapse.show, .collapse.collapsing'
`
We search for these selectors, but seems that some magic has to be done during `close` & `show`, and `.collapse` class is tweaked too. 

@WinterSilence  as I can understand you want to experiment over these,  so you can try remove  the 
```js
 this._element.classList.add(CLASS_NAME_COLLAPSE)

//and

 this._element.classList.remove(CLASS_NAME_COLLAPSE)

```

from code, and check the results :wink:

**Comment by WinterSilence** [comment]
@GeoSot I mean this flag activates not at begin of `show`/`hide` methods

**Comment by GeoSot** [comment]
yes, indeed. I did my checks based on your comments

 However is activated on proper timing, after the required checks, and before starting the transitioning process :smiley:

**Comment by WinterSilence** [comment]
@GeoSot 

> The problem seems to be there:
const SELECTOR_ACTIVES = '.collapse.show, .collapse.collapsing'

revert to `const SELECTOR_ACTIVES = '.show, .collapsing'`don't fixes bug too :(

**Comment by WinterSilence** [comment]
@GeoSot this problem detected in 5.0 too :( 
![изображение](https://user-images.githubusercontent.com/3521094/149959157-e32b84bb-8091-49f8-8dd9-5ac431cbdd1c.png)

**Comment by GeoSot** [comment]
`const SELECTOR_ACTIVES = '.collapse.show, .collapsing'`
plus something more

check this https://github.com/twbs/bootstrap/pull/35692/commits/b820965dfef2bf9fe262fc7606545f7b571c5cec

**Comment by genesi5** [comment]
I'm currently dealing with the same issue on Bootstrap 5.2, when switching between two Collapses (all toggled form button group). If i quickly toggle another tab (button in fact) before first collapse body is still in transition - both collapse bodies showing up. Also - it ruins button states, when it is shown as active, but its body collapsed.

https://user-images.githubusercontent.com/25093320/174462006-56d0a191-d558-4fca-8686-f32914de7c4d.mp4

**Comment by mdo** [comment]
v6-dev's accordion is now native `<details name>` (no JavaScript), and the shared `name` attribute makes the browser keep only one item open at a time — enforced synchronously, so rapid toggling can't leave multiple sections expanded. Closing as resolved in v6-dev — please open a new issue if you see multiple open on a native-`<details>` accordion there.
